config/schedule.rb

```diff
@@ -23,6 +23,11 @@
   rake "blacklight:delete_old_searches[7]"
 end
 
+# Runs the OpenDataHarvest DCAT script
+every :monday, at: "3:30am", roles: [:app] do
+  rake "uwm:opendataharvest"
```

Comment:
This will run the open data harvest script weekly.

After this runs a couple of times, I may up the frequency.

**This does not solve the problem of removing old files from opengeometadata, consider clearing the index periodically?**